### PR TITLE
feat(member_model): add airdropAddress to members_model

### LINKED ISSUE #11540: Add airdropAddress to the members model so that owners can airdrop to that address
### Description

Dependent on https://github.com/status-im/status-desktop/issues/11479

We need to show the `airdropAddress` of the members to the Owner. To do that, we need to add it to the members model in Nim.

### Acceptance Criteria

- The MembersModel now has a property for the `airdropAddress`

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #11540

One thing to note, if we end up fixing this issue https://github.com/status-im/status-desktop/issues/11573, we'll have to get the member address from somewhere else then the community description most likely.

The model wouldn't change though, so you can use this in QML.

### CODE REVIEW

**Review by caybro — APPROVED** [review]
LGTM

File: src/app/modules/main/chat_section/chat_content/users/controller.nim [hunk 1]
```diff
@@ -80,8 +80,6 @@ proc init*(self: Controller) =
   self.events.on(SIGNAL_LOGGEDIN_USER_IMAGE_CHANGED) do(e: Args):
     self.delegate.loggedInUserImageChanged()
 
-  let chat = self.getChat()
-
```
Comment (0x-r4bbit) [inline src/app/modules/main/chat_section/chat_content/users/controller.nim:84]: Was this just unused or how is this related?

File: src/app_service/service/chat/dto/chat.nim [hunk 2]
```diff
@@ -35,10 +35,16 @@ type
     large*: string
     banner*: string
 
+type RevealedAccount* = object
+  address*: string
+  chainIds*: seq[int]
```
Comment (0x-r4bbit) [inline src/app_service/service/chat/dto/chat.nim:40]: Do we really need `chainIDs` here?
Remember that those are currently only needed in status-go to check funds on given chains.
The airdrop address in itself is the same across chains, and I don't think we decide here on what chain the airdrop happens, or am I missing something?

**Review by mprakhov — APPROVED** [review]
LGTM

File: src/app/modules/main/chat_section/chat_content/users/controller.nim (hunk 1 above)
Comment (jrainville) [reply src/app/modules/main/chat_section/chat_content/users/controller.nim:84]: It was unused. I was looking for the place to edit the model so I was checking this controller and saw that this was useless. Just a clean up on the way.

File: src/app_service/service/chat/dto/chat.nim (hunk 2 above)
Comment (jrainville) [reply src/app_service/service/chat/dto/chat.nim:40]: I wasn't sure. I thought maybe on the frontend we might need to filter by chainID? Like if the owner only has funds in Optimism, it can't really airdrop on Arbitrum? I could be wrong though

Comment (0x-r4bbit) [reply src/app_service/service/chat/dto/chat.nim:40]: So these data are primarily for: on which chains does this member have funds to satisfy permission to join or view/post on a channel.

I think aidropping is rather decoupled from that. You could airdrop to members that don't have any funds anywhere.
Anyways, it surely doesn't hurt to have them here, but I don't think they are needed atm.
